flexmeasures/cli/utils.py

```diff
@@ -1,6 +1,13 @@
+from typing import Tuple, Mapping
```

Comment:
I have the impression that `Mapping` isn't meant to be used like this (from reading https://stackoverflow.com/questions/52487663/python-type-hints-typing-mapping-vs-typing-dict ). I suggest using `dict[str, str]` instead. And using `from __future__ import annotations` for Python 3.9 styled annotations.

Reply:
True, in this case, Dict makes for sense, but I tried it and `mypy` complained:

 ```
Argument 2 to "secho" has incompatible type "**Dict[str, str]"; expected "Optional[bool]"  [arg-type]
```